synergy-design-system/synergy-design-system
# chore: 🔧 Upgrade packages to latest versions

# Pull Request

## 📖 Description

This PR updates our internal packages to the latest stable version, according to our version ranges. It does not hold any breaking changes for package versions.

> It also adds a fix for the recent timeouts when using the e2e tests.

### 🎫 Issues

Closes #258 

## 👩‍💻 Reviewer Notes

Just check if everything is working fine. This means checking out the branch, installing, doing a clean build and making sure that tests and linting run fine.

## 📑 Test Plan

- Check out this branch
- Run a clean installation via `pnpm clean`
- Make sure vendorism works fine
- Make sure the builds work like they should
- Storybook should come up and have no issues when using it.

## ✅ DoD
